js/core/services/preload.service.js

```diff
@@ -81,6 +81,20 @@
                     document.body.classList.remove('jw-flag-loading-config');
                 });
 
+                // if flexbox is not supported show update browser page
+                var testElement           =    document.createElement('div');
```

Comment:
I think we should move this to a util function:

```
if (!utils.flexSupport()) {
  appStore.loading = false;

  return $state.go('updateBrowser', {directed: true});
}
```

Reply:
Agree

Reply:
Done